api/client/webclient/webclient.go

```diff
@@ -336,6 +336,12 @@ type AutoUpdateSettings struct {
 	ToolsVersion string `json:"tools_version"`
 	// ToolsMode defines mode client auto update feature `enabled|disabled`.
 	ToolsMode string `json:"tools_mode"`
+	// AgentVersion defines the version of teleport that agents enrolled into autoupdates should run.
+	AgentVersion string `json:"agent_version"`
+	// AgentAutoUpdate indicates if the requesting agent should attempt to update now.
+	AgentAutoUpdate bool `json:"agent_auto_update"`
```

Comment:
Noticing that this is inconsistent with the `tools_mode` field that has the same meaning.

@vapopov The `_mode` fields were intended for configuration, not for the command sent to clients/agents. Could we change `ToolsMode string` to `ToolsAutoUpdate bool` to match this PR?